src/codegen/llvm.zig

```diff
@@ -2647,7 +2647,7 @@ pub const FuncGen = struct {
         switch (struct_ty.zigTypeTag()) {
             .Struct => {
                 var ptr_ty_buf: Type.Payload.Pointer = undefined;
-                const llvm_field_index = llvmFieldIndex(struct_ty, field_index, target, &ptr_ty_buf);
+                const llvm_field_index = llvmFieldIndex(struct_ty, field_index, target, &ptr_ty_buf).?;
```

Comment:
I'm assuming here that `airStructFieldVal` should never be emitted when the field is zero sized, as then the result would always be comptime know, so the instruction would never make it to llvm.

Reply:
That assumption is sound :+1: